dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/DLCOracle.scala

```diff
@@ -234,7 +250,19 @@ case class DLCOracle(private val extPrivateKey: ExtPrivateKeyHardened)(implicit
     }
   }
 
-  def signEvent(
+  override def signEvent(
+      eventName: String,
+      outcome: DLCAttestationType): Future[EventDb] = {
```

Comment:
can't we just tighten this type to 

`outcome: EnumAttestation` so we know it's an enum attestation ?

Reply:
also maybe just rename the method to be `signEnumEvent()` as that is all it can do if i understand